Chp7Lsn1/Chp7Lsn1/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Chp7Lsn1
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] arrayEmpty = new int[20];
+
+            int [] displayArray = FillArray(arrayEmpty);
+            DisplayArray(displayArray);
+        }
+
+        public static int[] FillArray(int[] arrayEmpty)
+        {
+            for(int i = 0; i < 20; i++)
+            {
+                arrayEmpty[i] = i * 5;
+            }
+
+            int[] arrayFull = arrayEmpty;
+            return arrayFull;
+        }
+
+        public static void DisplayArray(int[] displayArray)
+        {
+            for (int mulFive = 0; mulFive < 20; mulFive++)
```

Comment:
Be careful with "chopped" naming like "mulFive" when perhaps you mean "multiplyFive".  Also, it isn't so great to put a constant value into the name of a variable.  You call it "mulFive", but what if requirements change so that you must start multiplying by 6?  The programmer should not have to rename variables to "mulSix" in order to make any adjustments.